regolith/helpers/a_presentationhelper.py

```diff
@@ -82,6 +83,9 @@ def subparser(subpi):
                        help="The database that will be updated.  Defaults to "
                             "first database in the regolithrc.json file.",
                        )
+    subpi.add_argument("-gc", "--google-calendar",
```

Comment:
I would make this default behavior, but allow it to fail gracefully (don't crash if it fails) but add a cli arg for --no-cal in case for some reason someone wants to explicitly turn it off.